devtools/test_dashboard/buttons.js

```diff
@@ -11,7 +13,7 @@ anchor.style.width = '1000px';
 
 function plotButtons(plots, figDir) {
 
-    Object.keys(plots).forEach( function (plotname) {
+    Object.keys(plots).forEach(function (plotname) {
```

Comment:
The major part of the plotly.js code has:

```js
Object.keys(plots).forEach(function(plotname) {
    // ...
});
```

eslint checks it using the `space-before-function-paren` rule: https://github.com/plotly/plotly.js/blob/eslint/.eslintrc#L25

which isn't considered an error (yet).